src/inventory.h

```diff
@@ -0,0 +1,29 @@
+/**
+ * @file inventory.h
+ * @author Austin Arlitt
+ */
+
+#ifndef INVENTORY_H
+#define INVENTORY_H
+
+// includes TBD
+
+/**
+ * A class for managing the inventory of a movie rental store
+ */
+class Inventory {
+public:
+    Inventory();                        // default constructor, creates empty inventory
```

Comment:
If this constructor doesn't do anything other than initialize member variables, consider member initialization instead.
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#Rc-default